# Dissecting epitranscriptional signatures by RNA editing in purified cortical cell types powers discovery and interpretation in bulk brain tissue
Ryn Cuddleston, Junhao Li, Xuanjia Fan, Alexy Kozenkov, Matthew Lalli, Stella Dracheva, Eran A. Mukamel, Michael S. Breen

Posttranscriptional modifications by RNA editing are pervasive in the brain, yet their cellular and genetic drivers remain poorly described. We quantified base-specific RNA editing across three major cell populations from the human prefrontal cortex: medial ganglionic eminence (MGE)-derived GABAergic neurons, glutamatergic neurons and oligodendrocytes. We uncovered a global increase in selective editing and RNA hyper-editing in neurons relative to oligodendrocytes. Bioinformatic analysis identified 189,229 bona fide cell type-associated sites spanning introns, 3â€™UTRs and exons, including sites implicated in neurodevelopment and neurological disorders. The cellular specificity for thousands of sites was confirmed by snRNA-sequencing. Importantly, cell type-associated sites were enriched in GTEx RNA-sequencing data, edited ~twentyfold higher than all other sites, and variation in RNA editing was predominately explained by neuronal proportions in bulk brain tissue. Finally, we discovered 661,791 cis-editing quantitative trait loci across thirteen brain regions, including hundreds with cell type-associated features. These data expand the repertoire of highly regulated RNA editing sites across cell types, brain regions and their genetic regulation. 


https://breenms.shinyapps.io/CNS_RNA_Editing
